api/config/imageUploader.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    'disk' => 'local',
+
+    'root' => 'files',
+
+    'branding' => [
+        'folder' => 'branding',
+    ],
+
+    'avatar' => [
+        'folder' => 'avatar'
+    ]
+];
```

Comment:
ok, but I did that in case we need to upload other type of files when we create events, and that allows us to use different configs for different file types images, docs etc

Reply:
@aussie-m-mike yeah, ok, `filesystems` config is a good place for this, you can put folder names there.